src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--styles calendar--start-day--mon сalendar--month-length--31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--mon сalendar--month-length--31">
```
CSS Класи і так використовуються для стилів